salt/tests/unit/modules/test_metalk8s_solutions.py

```diff
@@ -31,20 +31,8 @@ def test_virtual_success(self):
         """
         Tests the return of `__virtual__` function, success
```

Comment:
Minor: Maybe change the docstring and function name since this function always "success"ful